src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -321,6 +323,12 @@ private BracketChecker(String field) {
                 return Collections.singletonList(new IntegrityMessage(Localization.lang("should contain a four digit number"), entry, FieldName.YEAR));
             }
 
```

Comment:
We should add a comment to the BinteX doc here, because this decision on year structure is not intuitive.